client/a8c-for-agencies/style.scss

```diff
@@ -414,3 +414,18 @@ html {
 		}
 	}
 }
+
+.gray-tooltip {
+	.popover__arrow {
+		&::before {
+			border-bottom-color: var(--studio-gray-60) !important;
+			inset-block-start: 1px !important;
+		}
+	}
+	.popover__inner {
+		background: var(--studio-gray-60);
```

Comment:
Can we replace the studio color reference here with theme properties?

Reply:
Sorry, I copied this code from another place, so I missed this.